lib/resources/r4/related_person_patch.yaml

```diff
@@ -289,3 +289,19 @@ operations:
         <li>If no suffix value is provided, the suffix on the name will be unset.</li>
         <li>Only one suffix value may be provided.</li>
       </ul>
+
+  - name: remove-telecom
```

Comment:
Can we move this to be under `add-telecom`? I think the order could be add, test, remove.

Reply:
Will need to update r4_examples_relatedperson.rb to match the new order.

Reply:
Updated here: https://github.com/cerner/fhir.cerner.com/commit/e044c496534353b5646dd08dae7a0c036e2b6dc4